### PR TITLE
update for net6.0 and dependencies

### PR DESCRIPTION
**Pull request by jcmrva**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +8,5 @@ out
 AssemblyInfo.fs
 .DS_Store
 release.cmd
-analyzers
\ No newline at end of file
+analyzers
+.ionide
```
Comment (baronfel) [inline .gitignore:12]: with the most recent Ionide this shouldn't be required anymore - does the `.ionide` directory spawn with Ionide 6? (you'll have to delete it if it existed beforehand)

File: global.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "sdk": {
-        "version": "5.0.402"
+        "version": "6.0.100",
```
Comment (baronfel) [inline global.json:3]: mind bumping this to 6.0.200 at least? that's the most recent SDK feature band.

**Review by baronfel — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jcmrva) [reply .gitignore:12]: It doesn't, that was added via extension suggestion.
> You are missing entries in your .gitignore for Ionide-specific data files. Would you like to add them?

Comment (baronfel) [reply .gitignore:12]: oh lol, guess I can update that check in ionide itself now that it's no longer necessary.

**Review by baronfel — APPROVED** [review]

**Comment by baronfel** [comment]
Thanks for this!

**Pull request merged**